package/setup.py

```diff
@@ -67,6 +67,14 @@
     cython_found = False
     cmdclass = {}
 
+# Find our own release code (by shamelessly running MDAnalysis/version.py,
+#  which brings the __version__ variable into the current namespace.)
+with open("MDAnalysis/version.py") as f:
```

Comment:
Does this always work even when the package is installed in weird ways?  Ie is MDAnalysis/version.py always there?  Or would it be better to use resource_filename like we do with the test files?